lib/elevator.js

```diff
@@ -28,6 +28,7 @@
 
 var child_process = require('child_process');
 var commandBuilder = require('./command');
+var debug = require('debug')(require('../package.json').name);
```

Comment:
Yuck, is this 'normal' NodeJS syntax? I expect `require('../package.json').name` will never change, so for readability maybe it'd be better to just hard-code it?